Extensions/GoogleOptimize/Src/Tasks/GoogleOptimize/package-lock.json

```diff
@@ -0,0 +1,709 @@
+{
+  "name": "googleoptimize",
```

Comment:
Please add a Third Party notice for the modules that you are taking dependency on. @SumiranAgg  can help you on this.